mcfly.bash

```diff
@@ -60,8 +60,8 @@ PROMPT_COMMAND="mcfly_prompt_command;$PROMPT_COMMAND"
 #      render the search UI pre-filled with it.
 if [[ $- =~ .*i.* ]]; then
   if set -o | grep "vi " | grep -q on; then
-    bind "'\C-r': '\e0i#mcfly: \e\C-j mcfly search\C-j'"
+    bind "'\C-r': '\e0i#mcfly: \e\C-j mcfly search --results 10\C-j'"
```

Comment:
You can probably skip adding `--results 10` here since you have 10 as the default.

Reply:
True. My thinking when I amended that file was just to make it easier for the end-user using Ctrl-R (it would be clearer to know which part of the file to edit)

Reply:
I think most people will be using the mcfly.bash file out of an installation (like Homebrew) and shouldn't have to edit it directly.

Reply:
Oh, I didn't think about that.  I've reverted it now